core/src/auctions.ts

```diff
@@ -128,6 +130,34 @@ export const fetchAllAuctions = async function (network: string): Promise<Auctio
     return await Promise.all(auctionsWithMarketValue.map(a => enrichAuctionWithTransactionFees(a, fees)));
 };
 
+const enrichTakeEventWithDate = async function (network: string, takeEvent: TakeEvent): Promise<TakeEvent> {
+    const date = await fetchDateByBlockNumber(network, takeEvent.blockNumber);
+    return {
+        ...takeEvent,
+        date,
+    };
+};
+
+export const fetchTakeEvents = async function (network: string, auctionURL: string): Promise<Array<TakeEvent>> {
+    const { collateralType, auctionId } = parseAuctionURL(auctionURL);
```

Comment:
So `parseAuctionURL` (which can later be renamed to `parseAuctionId`) can throw an error. What happens then?

Hint: please validate that opening url with invalid parameter `INVALID` shows usual "no action found"

Reply:
I added a try catch that console logs the error if an invalid id is passed. The UI will then only show `No Auction found`.

I can also confirm that Auctions that do not exist are displayed with `No Auction found` as well.

Reply:
I mean code-wise there is a difference between:
- auctions that doesn't exist, ie inexistent auction index [`ETH-A:9000`](http://localhost:3000/collateral?network=mainnet&auction=ETH-A%3A9000)
- invalid auction index [`ETH-A:INVALID`](http://localhost:3000/collateral?network=mainnet&auction=ETH-A%3AINVALID)
- inexistent collateral type [`INVALID:1`](http://localhost:3000/collateral?network=mainnet&auction=INVALID%3A1)
- invalid auction id [`INVALID`](http://localhost:3000/collateral?network=mainnet&auction=INVALID)

Please test all edge cases by opening frontend with those urls

Reply:
> * auctions that doesn't exist, ie inexistent auction index [`ETH-A:9000`](http://localhost:3000/collateral?network=mainnet&auction=ETH-A%3A9000)

> * invalid auction index [`ETH-A:INVALID`](http://localhost:3000/collateral?network=mainnet&auction=ETH-A%3AINVALID)

> * inexistent collateral type [`INVALID:1`](http://localhost:3000/collateral?network=mainnet&auction=INVALID%3A1)

> * invalid auction id [`INVALID`](http://localhost:3000/collateral?network=mainnet&auction=INVALID)

All of them display `Auction Not Found`